src/components/atoms/LogoutFunction.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import Parse from "parse/dist/parse.min.js";
+import GenericButton from "./GenericButton";
+
+const LogoutButton = () => {
+  const navigate = useNavigate();
+
+  async function doUserLogOut() {
+    try {
+      await Parse.User.logOut();
+      // To verify that current user is now empty, currentAsync can be used
+      const currentUser = await Parse.User.current();
+      if (currentUser === null) {
+        return true;
+      } else {
+        alert(`Error! ${error.message}`);
+        return false;
+      }
+    } catch (error) {
+      alert(`Error! ${error.message}`);
+      return false;
+    }
+  }
```

Comment:
Is the Button responsible for the Log-Out process? 
What do you think of this approach, @mircealungu ?